website/content/docs/upgrading/upgrade-specific.mdx

```diff
@@ -61,25 +61,24 @@ This bug is fixed in Consul versions 1.13.1 and newer.
 
 #### Service mesh deployments using auto-encrypt or auto-config
 
-**Do not upgrade to Consul 1.13 yet** if using
+Upgrade to **Consul version 1.13.2 or later** if using
 [auto-encrypt](/docs/agent/config/config-files#auto_encrypt) or
 [auto-config](/docs/agent/config/config-files#auto_config).
 
-In Consul 1.13, auto-encrypt and auto-config both cause Consul
+In Consul 1.13.0 - 1.13.1, auto-encrypt and auto-config both cause Consul
 to require TLS for gRPC communication with Envoy proxies.
 In environments where Envoy proxies are not already configured
-to use TLS for gRPC, upgrading Consul 1.13 will cause
+to use TLS for gRPC, upgrading Consul Consul 1.13.0 - 1.13.1 will cause
 Envoy proxies to disconnect from the control plane (Consul agents).
 
-The underlying cause is the same as discussed in
-[deployments without the HTTPS port enabled on Consul agents](#service-mesh-deployments-without-the-https-port-enabled-on-consul-agents).
-However, when using auto-encrypt or auto-config,
-the problem **cannot** currently be avoided by 
-[modifying the agent's TLS configuration](#modify-the-consul-agent-s-tls-configuration)
-because auto-encrypt and auto-config automatically set
-interface-generic TLS configuration in a manner similar to
-[`tls.defaults`](/docs/agent/config/config-files#tls_defaults).
-We are working to address this problem in an upcoming 1.13 patch release.
+If upgrading to version 1.13.2 you must also enable
+[tls.grpc.use_auto_cert](/docs/agent/config/config-files#use_auto_cert)
+if you currently use the Connect CA and auto-encrypt certs for TLS
```

Comment:
Doesn't the "if you currently use the Connect CA" part apply to anyone using Consul service mesh? (Why is that part of the qualifier needed?)

Reply:
The idea is that non-mutual TLS requires: servers to provide a server certificate, and clients to trust the CA that signed the server certificate. 

So "Connect CA" is referring to the CA portion, and "auto-encrypt" certs to the server cert portion.

Though given that auto-encrypt requires the Connect CA, this could be rephrased as:
```suggestion
if you currently use the auto-encrypt certs as the server certs for TLS
```

What do you think?